web/src/views/CreateDandisetView/CreateDandisetView.vue

```diff
@@ -53,16 +56,21 @@ export default {
       return !(this.name && this.description);
     },
   },
+  mounted() {
+    if (!loggedIn()) {
+      this.$router.push({name: 'home'});
+    }
+  },
   methods: {
     async register_dandiset() {
       const { name, description } = this;
-      const { status, data } = await this.girderRest.post('dandi', null, { params: { name, description } });
+      const { status, data } = await girderRest.post('dandi', null, { params: { name, description } });
 
       if (status === 200) {
```

Comment:
Do we need this guard? I think Axios (and hence `girderRest`) will reject its promise (throwing an exception here) if there's a non-success status code.

Unless we can thoroughly recover from errors, I'd prefer not guarding against them at all, so they can bubble up, break the app in an obvious way, and be caught by Sentry.

Reply:
I verified that this does indeed break with a non-200 response code